relayer/middleware/source-tx.middleware.ts

```diff
@@ -40,11 +41,24 @@ export function sourceTx(
   optsWithoutDefaults?: SourceTxOpts,
 ): Middleware<SourceTxContext> {
   let opts: SourceTxOpts;
+  const alreadyFetchedHashes = new LRUCache({ max: 100 });
+
   return async (ctx, next) => {
     if (!opts) {
       // initialize options now that we know the environment from context
       opts = Object.assign({}, defaultOptsByEnv[ctx.env], optsWithoutDefaults);
     }
+    const vaaId = `${ctx.vaa.id.emitterChain}-${ctx.vaa.id.emitterAddress}-${ctx.vaa.id.sequence}`;
+    const txHashFromCache = alreadyFetchedHashes.get(vaaId) as
+      | string
```

Comment:
what is the union operator doing at the left of the type in this case?

Reply:
nothing, it's just `prettier` doing its formatting thing.